charts/consul/values.yaml

```diff
@@ -2138,7 +2138,7 @@ connectInject:
   # [injection annotation](https://developer.hashicorp.com/consul/docs/k8s/connect#consul-hashicorp-com-connect-inject)
   # to opt-in to Connect injection. If this is true, pods can use the same annotation
   # to explicitly opt-out of injection.
-  default: false
+  default: true
```

Comment:
Is this change required?

Reply:
nope, it was for local testing, reverting